CMakeLists.txt

```diff
@@ -40,40 +43,19 @@ endif()
 add_subdirectory(extlibs/minizip)
 add_subdirectory(extlibs/elzip)
 
-# snippet taken from
-# https://github.com/microsoft/vcpkg/blob/master/ports/sfml/usage
 find_package(SFML COMPONENTS system window graphics CONFIG REQUIRED)
 
+# snippet taken from
+# https://github.com/microsoft/vcpkg/blob/master/ports/sfml/usage
 if(OBE_USE_VCPKG)
-    include(FindLua)
-
-    find_package(lua REQUIRED)
-    find_package(sol2 CONFIG REQUIRED)
-    find_package(fmt CONFIG REQUIRED)
-    find_package(spdlog CONFIG REQUIRED)
-    find_package(Catch2 CONFIG REQUIRED)
-    find_path(TINYDIR_INCLUDE_DIRS "tinydir.h")
-
-
-    include_directories(${TINYDIR_INCLUDE_DIRS})
-
     if(WIN32)
         link_libraries(FLAC OpenAL OpenGL Vorbis)
```

Comment:
`target_link_libraries` :) ?

Reply:
It's on the main file, but I doubt it will need those checks and links.